AUTHORS.rst

```diff
@@ -1,3 +1,7 @@
+Authors
+=======
+
+Billinge Group and community contibutors.
```

Comment:
The main author was Sani Harouna-Mayer so let's mention him explicitly and add the group contributions after.  I see it is mentioned after, but maybe do the merge a bit differently so that it leads with his name, then it mentions group contributions.